### DISCUSSION TITLE
Versione 1.26.1

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#767] [Q&A] [answered]
Buongiorno,
nella versione in oggetto avete rilasciato "Generazione OTP per integrazione gestionale comunale".
Cercando cosa sia questa modifica non si trova nulla. L'unico documento modificato per la versione chagelog.md
come si nota

![image](https://github.com/italia/ansc/assets/127182190/fb8a52e4-1172-4b28-9cdd-b0adbe4fde4b)

Richiedo la documentazione necessaria

**Comment by mttfranci** [comment, marked as answer]
Buongiorno, in breve, la voce è nella sezione "Fixed" del changelog, quindi va letta come : 
"**Corretta** Generazione OTP per integrazione gestionale comunale" (il 12 c'è stato in effetti un problema sull'applicazione di generazione degli OTP per l'integrazione con i gestionali, rilevata da diversi comuni in produzione che abbiamo corretto).

NOTA: per maggiorni informazioni sul formato del changelog : il progetto aderisce al [Versionamento Semantico](https://semver.org/lang/it/) e se le linee guida per la struttura del changelog [tenere un changelog](https://keepachangelog.com/it-IT/1.1.0/).

Saluti

Assistenza ANSC
